src/character.cpp

```diff
@@ -2995,6 +2997,24 @@ ret_val<bool> Character::can_wear( const item &it, bool with_equip_change ) cons
         }
     }
 
+    if( it.has_flag( flag_TOURNIQUET ) ) {
+        bool need_tourniquet = false;
+        for( bodypart_id &bp : get_all_body_parts() ) {
+            if( !it.covers( bp ) ) {
+                continue;
+            }
+            if( get_effect_int( effect_bleed, bp->token ) > 9 && !worn_with_flag( flag_TOURNIQUET, bp ) ) {
```

Comment:
what's the magic number here?

Reply:
1/4 of max bleeding intensity, and I'll make it so in the code to remove the magic